packages/flutter/lib/src/widgets/scroll_activity.dart

```diff
@@ -613,7 +613,7 @@ class BallisticScrollActivity extends ScrollActivity {
   }
 
   @override
-  final bool shouldIgnorePointer;
+  bool shouldIgnorePointer;
```

Comment:
It does not look like this changes now, can it go back to being final?

Reply:
Of course, an oversight on my part.